src/docs/asciidoc/dev_env/kafka.adoc

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2018-2020 RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// This document is subject to the terms of the Creative Commons Attribution 4.0 International license.
+// If a copy of the license was not distributed with this
+// file, You can obtain one at https://creativecommons.org/licenses/by/4.0/.
+// SPDX-License-Identifier: CC-BY-4.0
+
+:kafka_schema: https://docs.confluent.io/current/schema-registry/index.html
+:confluent: https://www.confluent.io/
+:spring_kafka_doc: https://docs.spring.io/spring-kafka/reference/html/
+
+= Kafka Implementation
+
+Next to publishing cards to OperatorFabric using the REST API, OperatorFabric also supports publishing cards via a Kafka Topic. In the default configuration Kafka is disabled.
+
+
+== Enabling Kafka
+
+To enable Kafka support you need to set the `kafka.consumer.group_id property` in the cards-publication yml file:
+[source,yaml]
+----
+  kafka:
+    consumer:
+      group-id: opfab-command
+----
+
+Alternatively you can set the group id in the `config/docker/kafka-env.properties` file:
+[source, shell]
+----
+SPRING_KAFKA_CONSUMER_GROUP_ID=opfab-command
+----
+
```

Comment:
Er wordt uitgegaan van dat de brokers draaien op PLAINTEXT://127.0.0.1:9092 en dat er een schema registry draait op http://127.0.0.1:8081

Reply:
Check!. Heb nu ook spring.kafka.producer.bootstrap-servers  meegenomen in de settings.